lib/Ast.ml

```diff
@@ -2454,19 +2601,28 @@ end = struct
       | Pexp_sequence (lhs, rhs) -> exp_in_sequence lhs rhs exp
       | Pexp_apply (_, args)
         when List.exists args ~f:(fun (_, e0) ->
+                 match Extensions.Expression.of_ast e0 with
```

Comment:
What's going on here? Can you give an example in a comment?

Reply:
This doesn't seem any different than the usual `Extensions.Expression.of_ast` matches to me; what are you confused by?

Reply:
Not the `of_ast` specifically, but the `_ :: _` and `Yes, [e0]` bits. That is, it seems you're looking for some kind of source pattern to decide between `true` and `false` here for extensions, but the pattern you're looking for doesn't jump out at me.

Reply:
Look right below, at lines 2616–2619 – it's the same pattern as the existing code.  (To wit: bracketed collections that have attributes (and are equal to `exp`).)

Reply:
(I can update the comment, but it's `e0` because we can't compare with `ee0`, as that was synthesized.)

Reply:
OK. Yes. I see this now. A comment would be very helpful!

Reply:
Added in b32eff2